src/cohort.js

```diff
@@ -0,0 +1,36 @@
+class Cohort {
```

Comment:
good cohesion of classes, splitting the relevant methods into a Cohort class vs putting it all in the CohortManager class